pkg/config/fetcher/from_processes.go

```diff
@@ -23,9 +24,15 @@ func SecurityAgentConfig(config config.Reader) (string, error) {
 		return "", err
 	}
 
+	port := config.GetInt("security_agent.cmd_port")
+	if port <= 0 {
+		return "", fmt.Errorf("invalid security_agent.cmd_port -- %d", port)
+	}
+
 	c := util.GetClient(false)
+	c.Timeout = time.Second * 30
```

Comment:
Can you add a config option for this timeout ?

Reply:
I can, but I think it's a really narrow use case and don't think having a config option is very useful.

Reply:
At the end I added it ! 👍